Kernel/Bus/PCI/Device.h

```diff
@@ -35,4 +40,15 @@ class Device {
     Address m_pci_address;
 };
 
+template<typename... Parameters>
+ErrorOr<void> dmesgln_pci(Device const& device, AK::CheckedFormatString<Parameters...>&& fmt, Parameters const&... parameters)
```

Comment:
This shouldn't propagate error, if we fail to print messages, then we should just drop it and move on. This is best effort, not something that should fail kernel operations on failure (like OOM))

Reply:
Do you mean that we should  `release_error()` and ccontinue to return `void`? Let me know if the current changes look OK.

Reply:
What you have is closer to what I was thinking... In this specific case I think  if any of the `try_` API's fail we should return immediately (just bail out), but don't return an error. Logging is best case, and shouldn't cause a cascading failure.

Reply:
OK makes sense - I've updated the logger to return if we encounter any errors.